packages/shorebird_cli/lib/src/commands/release_new/aar_releaser.dart

```diff
@@ -0,0 +1,190 @@
+import 'package:archive/archive_io.dart';
+import 'package:io/io.dart';
+import 'package:mason_logger/mason_logger.dart';
+import 'package:path/path.dart' as p;
+import 'package:shorebird_cli/src/artifact_builder.dart';
+import 'package:shorebird_cli/src/code_push_client_wrapper.dart';
+import 'package:shorebird_cli/src/commands/release_new/release_type.dart';
+import 'package:shorebird_cli/src/commands/release_new/releaser.dart';
```

Comment:
nit but might be nice to export this as part of `commands/release_new` so we can use a single import

Reply:
Done.